### PR TITLE
Remove source from showcase

### PR DESCRIPTION
**Pull request by dipamsen**
## (Merge after #1063)

Removes "source" prop from all showcase files, following discussion from https://github.com/CodingTrain/thecodingtrain.com/issues/244#issuecomment-1153171288

### CODE REVIEW

**Comment by dipamsen** [comment]
@fturmel can you confirm if I have removed `source` from all the relevant places? (relevant commit: [2ec382f](https://github.com/CodingTrain/thecodingtrain.com/pull/1066/commits/2ec382f2f06f2232ec67943c30891a219681851f))

File: src/pages/index.js
Comment (fturmel) [inline src/pages/index.js:1]: remove videoId from destructuring (line 93)

**Review by fturmel — COMMENTED** [review]
Looks good to me, I think you got everything! Left a few comments on things that could additionally be removed. Caveat: I didn't review the json changes (too many to go through).

**Comment by shiffman** [comment]
Merging! We can always refine and make further changes later! Thank you both so much!

**Pull request merged**